actionview/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+*   Allow passing a class to `dom_id`.
+    You no longer need to call `new` when passing a class to `dom_id`.
+    This makes `dom_id` behave like `dom_class` in this regard.
+    Apart from saving a few keystrokes, it prevents Ruby from needing
+    to instantiate a whole new object just to generate a string.
+
+    Before:
+    ```ruby
+    dom_class(Post.new) # => "new_post"
+    dom_class(Post)     # => "new_post"
+    dom_id(Post.new)    # => "new_post"
+    dom_id(Post)        # => NoMethodError: undefined method `to_key' for Post:Class
+    ```
+
+    After:
+    ```ruby
+    dom_class(Post.new) # => "new_post"
+    dom_class(Post)     # => "new_post"
+    dom_id(Post.new)    # => "new_post"
+    dom_id(Post)        # => "new_post"
```

Comment:
We really only need the last example in this changelog since the behavior only changed for passing a class to `dom_id`.